roles/telemetry_verify_metrics/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+# tasks file for telemetry_verify_metrics
+- name: Verify OpenStack is deployed correctly
+  ansible.builtin.include_tasks:
+    file: openstack_deployed.yml
+  tags: precheck
+
+- name: Verify RabbitMQ metrics are being exposed and stored
+  ansible.builtin.include_tasks:
+    file: verify_rabbitmq_metrics.yml
+  tags: test
+  when: '"rabbitmq" in metric_sources_to_test'
```

Comment:
Role vars should start with the role name i.e. telemetry_verify_metrics_metric_sources_to_test